src/aspire/utils/blk_diag_func.py

```diff
@@ -140,50 +239,123 @@ def mat_to_blk_diag(mat, blk_partition):
 
 
 def blk_diag_transpose(blk_diag):
+    """
+    Get the transpose matrix of a block diagonal matrix
+
+    :param blk_diag: The block diagonal matrix
+    :return: The corresponding transpose form of a block diagonal matrix
+    """
+
     blk_diag_t = []
     for i in range(0, len(blk_diag)):
         blk_diag_t.append(blk_diag[i].T)
     return blk_diag_t
 
 
 def blk_diag_compatible(blk_diag_a, blk_diag_b):
+    """
+    Check whether two block diagonal matrices are compatible or not
+
+    :param blk_diag_a: The block diagonal A matrix
+    :param blk_diag_b: The block diagonal B matrix
+    :return: `True` value if all of shapes in two list representations of
+        block diagonal matrices are the same, otherwise return `False`.
+    """
     if len(blk_diag_a) != len(blk_diag_b):
         return False
-    elif np.allclose([np.size(mat_a, 1) for mat_a in blk_diag_a], [np.size(mat_b, 0) for mat_b in blk_diag_b]):
+    elif np.allclose([np.size(mat_a, 1) for mat_a in blk_diag_a],
+                     [np.size(mat_b, 0) for mat_b in blk_diag_b]):
         return True
     else:
         return False
 
 
 def blk_diag_isnumeric(x):
+    """
+    Check a block diag matrix is numeric or not
+    """
     try:
         return 0 == x*0
     except:
         return False
 
 
+def nonradial_filter2fb_mat(h_fun, fbasis):
```

Comment:
No need to specify `nonradial` here. It works for all filters, right? Also, let's avoid `2` for `to`. We can write it out. I propose `filter_to_fb_mat`.

Reply:
Done in the commit 1ed1cca.